lib/config/foldertype.go

```diff
@@ -30,12 +30,12 @@ func (t FolderType) MarshalText() ([]byte, error) {
 
 func (t *FolderType) UnmarshalText(bs []byte) error {
 	switch string(bs) {
-	case "readwrite":
-		*t = FolderTypeReadWrite
-	case "readonly":
-		*t = FolderTypeReadOnly
+	case "readwrite", "sendreceive":
+		*t = FolderTypeSendReceive
+	case "readonly", "sendonly":
+		*t = FolderTypeSendOnly
```

Comment:
My strong preference is to not change the on disk or JSON serialized config at this point. If we want to accept the synonyms as above in case someone writes them manually into the config i'm fine with that.

We will need to change the config format at the point another folder type is introduced. That will need to be a 0.15 release and it's fine to remove the old format and migrate it.